packages/components/psammead-radio-schedule/src/__snapshots__/index.test.jsx.snap

```diff
@@ -1,112 +1,102 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`RadioSchedule should render ltr radio schedules correctly 1`] = `
-.emotion-161 {
+.emotion-1 {
   padding: 0;
   margin: 0;
 }
 
 @media (max-width:14.9375rem) {
-  .emotion-161 {
+  .emotion-1 {
     padding: 0 0.5rem;
   }
 }
 
 @media (min-width:15rem) and (max-width:24.9375rem) {
-  .emotion-161 {
+  .emotion-1 {
     padding: 0 0.5rem;
   }
 }
 
 @media (min-width:25rem) and (max-width:37.4375rem) {
-  .emotion-161 {
+  .emotion-1 {
     padding: 0 1rem;
   }
 }
 
 @supports (display:grid) {
-  .emotion-161 {
+  .emotion-1 {
     display: grid;
     position: initial;
+    width: initial;
+    margin: 0;
   }
 
-@media (max-width:14.9375rem) {
-    .emotion-161 {
-      width: initial;
-      margin: 0;
```

Comment:
Emotion seems to be optimising this for us? These two lines appear 6 times (once in every media query) in the "before"

In the after, they are just specified once, outside any media query

Accounts for 10 of the 40 removed lines